ras_backstage/resources/sign_in_uaa/sign_in_uaa.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from flask import request, make_response, jsonify
+from flask_restplus import fields, Resource
+from structlog import wrap_logger
+
+from ras_backstage import sign_in_uaa_api
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+sign_in_details = sign_in_uaa_api.model('SignInDetails', {
+        'username': fields.String(required=True, description='username'),
+        'password': fields.String(required=True, description='password')
+})
+
+
+@sign_in_uaa_api.route('/', methods=['POST'])
+class SignInUaa(Resource):
+
+    @staticmethod
+    @sign_in_uaa_api.expect(sign_in_details, validate=True)
+    def post():
+        logger.info('Retrieving sign-in details')
+        # force=true means the post doesn't HAVE to have application/json
+        # in its content type.  This feels wrong to me but is also in the django signin code...
```

Comment:
Pleased remove this comment, either don't use force=True and handle the None reponse yourself or leave as is and use the on_json_loading_failed function instead. The default implementation will raise a bad request if the request body can't be parsed to json

http://flask.pocoo.org/docs/0.12/api/#flask.Request.get_json

Reply:
There should be no need to be using `force=True` here anyway. We're in control of the only place which will make a request to here so we know what form it's coming in as

Reply:
Agreed

Reply:
I agree with @andrewmil - I'll take it out.